src/main/groovy/grails/plugin/sentry/SentryConfig.groovy

```diff
@@ -95,6 +93,25 @@ class SentryConfig {
             sanitizeStackTrace = true
         }
 
+        if (config.inAppIncludes) {
+            if (config.inAppIncludes instanceof List) {
+                inAppIncludes = config.inAppIncludes as List
+            }
+            if (config.inAppIncludes instanceof String) {
+                inAppIncludes = (config.inAppIncludes as String).split(",") as List
```

Comment:
if we support a csv, do we care about whitespace trimming?

Reply:
This is the way how lists are handled in this class so I used the same for consistency. Actually I don't like it much at all and would rather use standard `@ConfigurationProperties(prefix = "grails.plugin.sentry")`, but there was problem with accessing this bean in [SentryGrailsPlugin.doWithSpring()](https://github.com/agorapulse/grails-sentry/blob/781a7d7c28eb2aea2bcfac6efdfd3e2c85e1179b/src/main/groovy/grails/plugin/sentry/SentryGrailsPlugin.groovy#L63).